pkg/controller/glustercluster/glustercluster_controller.go

```diff
@@ -96,6 +96,25 @@ func (r *ReconcileGlusterCluster) Reconcile(request reconcile.Request) (reconcil
 		return reconcile.Result{}, err
 	}
 
+	// ----- general flow of reconcile -----
+
+	// Get current reconcile version from CR
+
+	// If no current version, use highest version to reconcile
+	// otherwise, choose the highest compatible version
```

Comment:
Also inside "glustercluster" we could have "controller.go" instead of "glustercluster_controller.go"

Reply:
It's sdk-chosen... not ideal, but :man_shrugging: